docker/README.md

```diff
@@ -0,0 +1,254 @@
+# Nuxeo Docker Images
+
+Nuxeo provides a set of Docker images that can be separated into two categories:
+- The [builder](#builder-images) images.
+- The [final](#final-images) images.
+
+The builder images allow to build a final Docker image using a multi-stage build.
+
+The final images are ready to use.
+
+These images are pushed to our [public Docker regsitry](https://packages.nuxeo.com/#browse/search/docker). To pull an image, run:
+
+```
+docker pull docker.packages.nuxeo.com/IMAGE_NAME:TAG
+```
+
+For instance, to pull the latest 11.1-SNAPSHOT version of the `nuxeo/nuxeo` image, run:
+
+```
+docker pull docker.packages.nuxeo.com/nuxeo/nuxeo:11.1-SNAPSHOT
+```
+
+## Disclaimer
+
+These Docker images don't aim to replace the [Nuxeo Docker official image](https://hub.docker.com/_/nuxeo/). They implement a different approach, described [here](https://jira.nuxeo.com/browse/NXP-27514), to try having immutable images configured at build time instead of runtime.
+
+## Builder Images
+
+The versioning of these images follows the Nuxeo platform versioning. In the Dockerfile samples below, `VERSION` can be replaced for instance by `11.1-SNAPSHOT`.
+
+### Base Builder: nuxeo/builder
+
+It provides:
+- A Nuxeo server distribution with appropriate permissions in the `/distrib` directory.
+- An `install-packages.sh` script to install Nuxeo packages.
+
+It is based on an OpenJDK image as it requires Java for the package installation.
+
+It must be used within a multi-stage build.
+For instance, you can use the following Dockerfile sample to build an image based on a `BASE_IMAGE`, containing a Nuxeo server distribution:
+- In the `NUXEO_HOME` directory.
+- With some Nuxeo packages installed, whose ZIP files must be located in the `local/packages` directory.
+- Owned by the UID user and GID group.
+
+```
+FROM nuxeo/builder:VERSION as builder
+COPY local/packages /packages
+RUN install-packages.sh /packages
+
+FROM BASE_IMAGE
+COPY --from=builder --chown=UID:GID /distrib NUXEO_HOME
+``` 
+
+### Platform Builder: nuxeo/builder-platform
```

Comment:
why builder-platform and not maven-builder?

Reply:
The "platform" part refers to the future "Content Platform" image this builder is done for, not existing yet, see https://jira.nuxeo.com/browse/NXP-28133.
Meaning it's a builder like the basic on (nuxeo/builder) + some Nuxeo packages.

Reply:
ok so it is not a generic builder but a dedicated one to build the final platform image?

Reply:
Yes, though it can still be considered as "generic" in the sense that it still allows to build an image starting form a Nuxeo distrib and a predefined set of packages (that's the builder-platform one) + whatever you want to add in your multi-stage build: additional Nuxeo packages in the build stage, additional OS packages in the target stage, ...